openslides_backend/migrations/migrations/0038_remove_user_default_projector.py

```diff
@@ -38,23 +38,17 @@ def migrate_event(
         self,
         event: BaseEvent,
     ) -> Optional[List[BaseEvent]]:
+        events = super().migrate_event(event)
+        if events is None:
+            return events
```

Comment:
This should never happen, but you should nevertheless still use the old event in this case to fully execute the migration.

Reply:
As far as I understand the code of RemoveFieldsMigration this case can happen, if collection doesn't match with one collection in the dict.

Reply:
Then I want to stop processing anymore.

Reply:
Right! My bad